templates/sensu.erb

```diff
@@ -1,6 +1,10 @@
 EMBEDDED_RUBY=<%= scope.lookupvar("sensu::use_embedded_ruby") %>
 LOG_LEVEL=<%= scope.lookupvar("sensu::log_level") %>
+<% if scope.lookupvar("sensu::rubyopt") -%>
```

Comment:
at least in puppet 3.8 these will return :undef  perhaps  `if scope.lookupvar("sensu::rubyopt") == :undef` ??